src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -415,7 +415,20 @@ namespace winrt::TerminalApp::implementation
     {
         if (const auto& realArgs = actionArgs.ActionArgs().try_as<TerminalApp::CloseOtherTabsArgs>())
         {
-            uint32_t index = realArgs.Index();
+            uint32_t index;
+            if (realArgs.Index())
+            {
+                index = realArgs.Index().Value();
+            }
+            else if (auto focusedTabIndex = _GetFocusedTabIndex())
+            {
+                index = *focusedTabIndex;
+            }
+            else
+            {
+                actionArgs.Handled(true);

```

Comment:
Should we actually mark this event as handled? I think usually in a scenario like this we mark it as `Handled(false)`

Reply:
Agreed.... if you can't actually do something with the data you're given... it's `Handled(false)`.